test/logger.test.js

```diff
@@ -1480,3 +1480,15 @@ test('should not log incoming request and outgoing response when disabled', t =>
     t.is(lines[0].msg, '500 error')
   })
 })
+
+test('should pass when using unWritable props in the logger option', t => {
+  t.plan(1)
+  try {
+    Fastify({
+      logger: Object.defineProperty({}, 'level', { value: 'info' })
+    })
+    t.pass()
```

Comment:
What do you think to test if this works with: `Object.defineProperties({}, { level: 'info', stream: streamExample })` like the first test suite?

Reply:
The above test case check object assignment value when the  object's properties unWritable, I think the 'level' or 'stream' is same